src/HttpPartialResponse.php

```diff
@@ -21,6 +21,7 @@ final class HttpPartialResponse implements PartialResponse
     private Headers $headers;
 
     /**
+     * @param HttpPartialResponse[] $lines
```

Comment:
The `$lines` param is actually `string[]`

Reply:
Oh no! It proves I'm no developer (but a janitor)

Fixing the fix .... :)

Reply:
Oh no Again :)

It was already fixed in the third commit.

Reply:
Oh sorry, I probably wrote the comment before looking at the other commit.